### PR TITLE
Update SORACOM-LPWAN-with-features.csv

### PR DESCRIPTION
**Pull request by MarPiSora**
July 2024 update

### CODE REVIEW

**Review by MarPiSora — COMMENTED** [review]
looks good now

**Comment by MarPiSora** [comment]
hello @fstubner - may I ask for a quick review?
No major changes...just some new networks and aupdated timestamps to show.

File: SORACOM-LPWAN-with-features.csv [hunk 1]
```diff
@@ -132,23 +136,27 @@ Israel,ISR,42502,Cellcom,LTE-M,B3,plan01s,Validated,PSM and eDRX | SMS,01/30/202
 Israel,ISR,42501,Partner,LTE-M,B3,plan01s,Validated,PSM and eDRX | SMS,01/30/2024
 Italy,ITA,22210,Vodafone,LTE-M,B3 | B20,plan01s,Validated,PSM* and eDRX | SMS,01/30/2024
 Italy,ITA,22210,Vodafone,LTE-M,B3 | B20,planX3,Validated,eDRX* (81.92) and PSM* | SMS,06/06/2024
+Japan,JPN,44052,KDDI,LTE-M,B18,plan-KM1,Guaranteed,PSM* and eDRX | SMS,01/12/2024
+Japan,JPN,44051,KDDI,LTE-M,B18,plan01s,Validated,PSM | SMS,07/01/2024
```
Comment (zelkovab) [inline SORACOM-LPWAN-with-features.csv:140]: Could you remove plan01s KDDI (44051) from the list?
We know the network attach itself works but we need to hide it from the political reasons..

**Review by zelkovab — CHANGES_REQUESTED** [review]
added one comment for KDDI(44051) x 01s. the rest looks good to me.

**Comment by MarPiSora** [comment]
removed KDDI plan01s LTE-M
-> adapted this is test sheet as well to be not shown anymore in the future

**Review by zelkovab — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**